marklogic-data-hub-central/ui/src/util/environment.tsx

```diff
@@ -32,4 +30,8 @@ export function getEnvironment():any {
     }
 }
 
-export default { setEnvironment, getEnvironment };
+export function resetEnvironment() {
```

Comment:
When would this be called? I don't understand the concept of "reset environment" in HC on DHS, as the user has no control over the environment.

Reply:
The function resetEnvironment() ensures the project name header link (currently hardcoded to "Data Hub Project") in the UI is not displayed when user logs out.

Reply:
That name will soon be the DHS service name, which is intended to be a meaningful identifier to the user. I think it makes sense to have that displayed in the login page so the user knows which DHS instance they're logging into. Seems like a requirements question, check with @sbayatpur .

Reply:
Yeah, that's a different story that will probably taken up next sprint. In this PR, I am just adding back the snippet I had removed inadvertently in my previous PR for removing unwanted code from HC